rest/taskrouter/statistics/workers/cumulative/example-1.4.x.js

```diff
@@ -8,7 +8,7 @@ const client = require('twilio')(accountSid, authToken);
 
 client.taskrouter.v1
   .workspaces(workspaceSid)
-  .workers()
+  .workers
```

Comment:
Hi @JenniferMah , the 1.3.x example where we use `workers()` or `workers('WKXXX')` works for me when testing but not this one. Is there a reason why that is no longer required in 1.4?

Reply:
Hi @bermudezmt! In the version 4.x.x of twilio-node we [removed support for a WorkerSid for Cumulative and Real-Time Workers Statistics](https://github.com/twilio/twilio-node/pull/820). We have documented this change in the [upgrade guide](https://github.com/twilio/twilio-node/blob/main/UPGRADE.md?plain=1#L46).

Reply:
Thank you so much!